catalog-entities/marketplace/plugins/mcp-tools.yaml

```diff
@@ -94,7 +95,7 @@ spec:
          disabled: false
       
        # MCP tools plugins
-       - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-software-catalog-mcp-tool:bs_1.42.5__0.2.3!red-hat-developer-hub-backstage-plugin-software-catalog-mcp-tool
+       - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-software-catalog-mcp-tool:next__0.4.0!red-hat-developer-hub-backstage-plugin-software-catalog-mcp-tool
          disabled: false
        - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-techdocs-mcp-tool:bs_1.42.5__0.3.0!red-hat-developer-hub-backstage-plugin-techdocs-mcp-tool
```

Comment:
do you need to also bump this plugin from 1.42 to 1.45?

Reply:
Yes, it should be `bs_1.45.3__0.3.2`

Reply:
if you want to bump NOW you can get `next__` tag

if you want to wait until after Jan 13 when we have the `release-1.9` branches created, then you'll also get the new `bs_1.45.3__` tags instead.

And if you think it's annoying to have this distinction and double-maintenance load (update now, update again in Jan) then please vote for https://github.com/redhat-developer/rhdh-plugin-export-overlays/pull/1551 so I can convince people that next tags are kinda pointless and just introduce extra work.

Reply:
Jordan's off now til the new year, but #1551 gets my vote :)